pygmyui/templates/pygmy/dashboard.html

```diff
@@ -30,6 +30,7 @@
       <td>{{ link.created_at }}</td>
       <td>{{ link.hits_counter }}</td>
       <td>
+        <a download="qr-code-{{ link.short_code }}" href="data:image/png;base64,{{ link.qr_code }}" class="stat-button">QR-Code</a>
```

Comment:
Why not make it a button element as well for some more consistency? 🤔

Reply:
HTML buttons can't have a href attribute thus can't generate the download file from the base64 string.